srcpkgs/swaylock-effects/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'swaylock-effects'
+pkgname=swaylock-effects
+version=1.6
+revision=3
+wrksrc=${pkgname}-${version}-${revision}
```

Comment:
Revision is voids internal number to bump the package if a rebuild is required, i.e. because a dependency is updated.
This revision is part of the version number, i.e use something like:

```
version=1.6p3
wrksrc="${pkgname}-${version/p/-}"
distfiles="https://github.com/mortie/${pkgname}/archive/v${version/p/-}.tar.gz"
```

Reply:
Thank you for your help. I really appreciate it. :smiley: